### PR TITLE
Feature/multibuilding

### PR DESCRIPTION
**Pull request by ThibaultMarzullo**
New features:
- new makefile can start several workers at once
- new client can manage several building simulations at once
Solving [Issue #29](https://github.com/henze-research-group/MODRLC/issues/29).

### CODE REVIEW

**Comment by ThibaultMarzullo** [comment]
@nllong have you had time to review this?

File: actb_client/jobs.json [hunk 1]
```diff
@@ -1 +1 @@
-{"spawnrefsmalloffice": [{"simId": "384417fc-fbbb-40e7-89af-0cfcfabbf536", "url": "http://127.0.0.1:80", "placeholder": null}]}
\ No newline at end of file
+{"spawnrefsmalloffice": [], "spawnrefmediumoffice": [], "spawnrefsmalloffice-variant1": []}
```
Comment (nllong) [inline actb_client/jobs.json]: Just adding in a final line feed at the end of the file
```suggestion
{"spawnrefsmalloffice": [], "spawnrefmediumoffice": [], "spawnrefsmalloffice-variant1": []}

```

File: examples/python/BuildingPortfolio/3SmallOffices.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            'forecasting' : {'horizon' : 84600, 'interval' : 300},
+            'step' : 300,
+            'metamodel' : None
+            } #todo add scenario
+
+# We need to give a unique ID for each building using a dictionary as shown below.
+# Here, we simply use 3 identical Small Office buildings for demonstration purposes, named 'first',
+# 'second' and 'third'.
+
+buildings = {'first' : basicSOM,
```
Comment (nllong) [inline examples/python/BuildingPortfolio/3SmallOffices.py:24]: this is great!

File: examples/python/BuildingPortfolio/3SmallOffices.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+for i in range(100):
+    for id in buildings.keys():
+        # We pass the id of the building we want to advance, and the control vector
+        client.advance(id, u)
+
+# Similarly to the advance method, we retrieve KPIs at the end of a simulation run.
+
+for id in buildings.keys():
+    print("KPIs for building named {}:".format(id))
+    print(client.kpis(id))
```
Comment (nllong) [inline examples/python/BuildingPortfolio/3SmallOffices.py]: Just adding in a final line feed at the end of the file
```suggestion
    print(client.kpis(id))
    
```

**Review by nllong — APPROVED** [review]
💥🏆

**Pull request merged**